src/components/Events/GM/GM.js

```diff
@@ -198,10 +161,10 @@ function GM_COUNTDOWN(props) {
 					<div className="countdown-labels">{props.secondString}</div>
 				</div>
 			</div>
-			<h3 id="gm-zoom-release">Zoom link, slides, and other resources will be provided on Thursday, January 7th, 2021 at 6:30 p.m. PST</h3>
-			<p id="posted-to-youtube">The Zoom portion of Winter General Meeting 2021 will be recorded and posted to our <a href="https://www.youtube.com/c/acmucla" target="_blank" rel="noopener noreferrer">YouTube channel</a>.</p>
-			<p id="cant-wait">Can&rsquo;t wait to join in on the fun? Jump in right now with our <a href="https://linktr.ee/acmucla" target="_blank" rel="noopener noreferrer">Linktree</a> or hop into the official <a href="https://bit.ly/ACMdiscord" target="_blank" rel="noopener noreferrer">Discord</a> and say hello!</p>
-			<img id="gm-graphic" src={process.env.PUBLIC_URL + '/images/Winter_GM_2021_graphic.png'} alt="Winter General Meeting 2021 graphic" />
+			<h3 id="gm-zoom-release">Zoom link, slides, and other resources will be provided on {GM_START_TIME.toLocaleString('en-US', {weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' })} at {gmData.startTime} {gmData.times.timezone}</h3>
```

Comment:
Not a huge issue, but you can consider moving the function call to `toLocaleString` into a separate helper function. (i.e. this bit `GM_START_TIME.toLocaleString('en-US', {weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' })`)

In my personal experience, moving as much complicated logic (or javascript expressions/statements that take a moment to read) to dedicated functions helps with code separation. 

In brief, you want to keep as much of our "template" (html) separate from our "controlling logic" that actually generates the dynamic content.